SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005746	Plant a flag that raises the combat abilities of the pirate crew. A combo is activated if multiple enemies is defeated around the jolly roger. Also at certain rate, attacking can pillage items from enemies and the rate is increased as Jolly Roger level is raised. (Combo is only activated if the party leader is Corsair.)
+SKILL_20151224_005747	Pistol Shot
+SKILL_20151224_005748	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire continuous shots with pistol to damage multiple enemies ahead.
+SKILL_20151224_005749	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Slash enemies with powerful swing to damage the enemy.
+SKILL_20151224_005750	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Spin the weapon to continuously damage the enemy in front of you.
+SKILL_20151224_005751	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Damage the enemy by slashing multiple times from side to side.
+SKILL_20151224_005752	Attack enemies in the area by exploding the gunpowder in hand. This technique will also damage the user and might cause combat incapability if the HP is too low.
+SKILL_20151224_005753	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: %{CaptionRatio}#%{nl}HP consumption: #{CaptionRatio2}#
+SKILL_20151224_005754	Creates clones of yourself. Clones will act along with the main body and skills can be used along with the clones. HP of the clones is determined by the user's dexterity.
+SKILL_20151224_005755	Duration: #{CaptionRatio}#second{nl}Clone summon count: #{CaptionRatio2}#
+SKILL_20151224_005756	Counter-attack by tricking the enemy with fake doll created from a log.
+SKILL_20151224_005757	Attack multiple enemies by continuously exploding gunpowder in hand.
+SKILL_20151224_005758	Attack multiple enemies by throwing kunais.
+SKILL_20151224_005759	Dragontooth
+SKILL_20151224_005760	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Rotate your spear using centrifugal force to attack the enemy.
+SKILL_20151224_005761	Serpentine
+SKILL_20151224_005762	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Use your Spear to deal continuous damage to enemies in front of you.
+SKILL_20151224_005763	Gae Bulg
+SKILL_20151224_005764	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Throw your spear forward to attack multiple enemies. The thrown spear must be recovered after skill use.
+SKILL_20151224_005765	Dragon Soar
+SKILL_20151224_005766	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Vault into the air while swinging your spear upward. Enemies hit with the skill are temporarily launched.
+SKILL_20151224_005767	Summon Guild Member
+SKILL_20151224_005768	Summon designated guild member to their current location. Need an approval from the designated guild member. Can't summon guild members in an instance dungeon.
+SKILL_20151224_005769	Guild member summon count: #{CaptionRatio}#
+SKILL_20151224_005770	Warp To Guild Member
+SKILL_20151224_005771	Warp to designated guild member's location. Can't warp to guild members in an instance dungeon.
```

Comment:
Warps to a specific guild member's location. You cannot warp to guild members who are in an instanced dungeon.